### PR TITLE
Saas 15.1 services bug fix kcv

### PR DESCRIPTION
**Pull request by kcv-odoo**
This PR fixes various bugs in seas-15.1 for the services apps.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by xavierbol — APPROVED** [review]
LGTM, thanks for the fix. 🙂

File: addons/project/views/project_views.xml [hunk 1]
```diff
@@ -360,14 +360,6 @@
                                 <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color', 'no_quick_edit': 1}"/>
                             </div>
                         </button>
-                        <button class="oe_stat_button" type="object" name="action_view_analytic_account_entries" icon="fa-usd" attrs="{'invisible': [('analytic_account_id', '=', False)]}" groups="analytic.group_analytic_accounting">
```
Comment (xavierbol) [inline addons/project/views/project_views.xml:363]: See with @IT-Ideas , I'm not sure if we can remove the button in stable even if it is duplicated. 🤔
For me, I think it is good and it should not break something. 😇

**Review by xavierbol — APPROVED** [review]
LGTM, thanks. I just let a comment for LTU when he will review it ;)

File: addons/project/views/project_views.xml (hunk 1 above)
Comment (IT-Ideas) [reply addons/project/views/project_views.xml:363]: Indeed as it is duplicated and as they have the same attributes, we can assume that xpath will not break.

**Comment by IT-Ideas** [comment]
I just changed a bit the commit messages.

**Review by IT-Ideas — APPROVED** [review]
LGTM Thanks 🙂

**Comment by IT-Ideas** [comment]
@robodoo rebase-ff r+

**Pull request closed without merging**